config.m4

```diff
@@ -257,11 +257,11 @@ if test "$PHP_MONGODB" != "no"; then
       AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
       AC_MSG_CHECKING(for libmongocrypt)
 
-      if test -x "$PKG_CONFIG" && $PKG_CONFIG --exists libmongocrypt-1.0; then
-        if $PKG_CONFIG libmongocrypt-1.0 --atleast-version 1.0.1; then
```

Comment:
libmongocrypt-1.0 doesn't exists

Reply:
Thanks for catching this! I suppose this is another difference between libmongocrypt and libmongoc's packaging. /cc @kevinAlbs

Reply:
This is not related to packaging, but to what upstream .pc file defines

Reply:
Sorry, I didn't mean to imply this was OS-specific packaging that you handle. I assume the issue is that libmongocrypt's `.pc` file doesn't include "-1.0" in the name, which is inconsistent with libmongoc and libbson.

As an aside, there is also a `${PROJECT_NAME}` in [`libmongocrypt.pc.in`](https://github.com/mongodb/libmongocrypt/blob/master/cmake/libmongocrypt.pc.in), but I'm not sure where that is set. `PROJECT_VERSION` is assigned in [`CMakeLists.txt`](https://github.com/mongodb/libmongocrypt/blob/1.0.1/CMakeLists.txt#L304), but `PROJECT_NAME` is nowhere to be found. I assume that is just defaulting to the base name of the `.pc` file or perhaps the build directory.